src/redux/actions.js

```diff
@@ -0,0 +1,66 @@
+import {
+  GET_ALL,
+  GET_SINGLE,
+  SEARCH_CASES,
+  SEARCH_REGIONS,
+} from './actionType';
+import * as API from './api';
+
+export const getAllCases = () => async (dispatch) => {
+  const { data } = await API.retrieveCases();
+  const {
+    dates: {
+      '2020-03-22': { countries },
+    },
+  } = data;
+
+  const selectedValues = Object.values(countries).map((item) => ({
+    id: item.id,
+    name: item.name,
+    regions: item.regions,
+    today_confirmed: item.today_confirmed,
+  }));
+
+  // const withRegions = Object.values(selectedValues).filter(
+  //   (region) => region.regions.length !== 0,
+  // );
```

Comment:
Please remove commented code as it can make the whole code base unclean.

Reply:
Alright. Thank you @SajjadAhmad14  I'm on it already